app/views/posts/_write_post.haml

```diff
@@ -41,6 +43,18 @@
         = f.hidden_field :icon_id, id: 'reply_icon_id'
         = submit_tag (method == :put ? 'Save' : "Post"), class: 'button', id: 'submit_button', disable_with: 'Saving...'
         = submit_tag "Preview", class: 'button', id: 'preview_button', name: 'button_preview'
+      %br
+      #markup-allowed-box{style: 'display:none'}
```

Comment:
Is this complex enough to be worth its own partial in case we change it later? No PR description here but I vaguely recall we might want to UI it slightly more later.